components/policies/development/konflux-rbac/konflux-support-viewer-access/generate-support-viewer-rolebinding-clusterpolicy.yaml

```diff
@@ -0,0 +1,59 @@
+---
+# This ClusterPolicy automatically generates a RoleBinding in all namespaces
+# to grant read-only access to the 'konflux-sre' and 'ai-konflux-user-support' groups.
+#
+# This policy is designed to ensure that these groups have consistent
+# visibility across the cluster for monitoring, troubleshooting, or support.
+
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: generate-konflux-support-read-only-rolebinding
+  annotations:
+    policies.kyverno.io/title: "Generate Read-Only RoleBinding for Konflux support and sre Groups"
+    policies.kyverno.io/category: Multi-Tenancy
+    policies.kyverno.io/description: >-
+      This policy automatically generates a RoleBinding in all namespaces.
+      The RoleBinding binds the 'konflux-sre' and 'ai-konflux-user-support'
+      groups to the Konflux-specific 'konflux-viewer-user-actions' ClusterRole,
+      granting them comprehensive read-only access to resources within each namespace,
+      therefore allowing better, fast and streamlined support.
+spec:
+  background: true
+  rules:
+    - name: generate-read-only-rolebinding
+      match:
+        any:
+        - resources:
+            kinds:
+            - Namespace
+      exclude:
+        any:
+        - resources:
+            namespaces:
+            - kube-system
+            - kube-public
+            - kube-node-lease
+            - kyverno
+            - openshift-monitoring
+            - openshift-ingress
+            - openshift-operators
```

Comment:
I presume you're only wanting to do this on tenant namespaces, right?  You may want to match only namespaces that have the `konflux-ci.dev/type=tenant` label.

The policy `bootstrap-tenant-namespace-queue` has an example of how to do this (among other policies).